main.js

```diff
@@ -0,0 +1,91 @@
+document.getElementById('submit').addEventListener('click', function () {
```

Comment:
Would not recommend putting everything under the event listener like this - better to separate out into reusable functions